### PR TITLE
Pandas reader improvements

### PR DESCRIPTION
**Pull request by s-ducks**

### CODE REVIEW

File: .version [hunk 1]
```diff
@@ -0,0 +1 @@
+0.5.0
```
Comment (isichei) [inline .version:1]: Not easier just to hard code this in the `__init__.py`?

File: arrow_pd_parser/export.py [hunk 2]
```diff
@@ -77,25 +80,30 @@ def pd_to_json(
...
     Args:
         df (pd.DataFrame): a pandas dataframe
         output_file (str): the path you want to export to (s3)
         from_pandas_kwargs (optional, dict): kwargs to pass to pyarrow.Table.from_pandas
         write_table_kwargs (optional, dicr):
             kwargs to pass to pyarrow.parquet.write_table
-        arrow_schema (optional, pyarrow.lib.schema): schema to cast the dataframe to
-            during conversion
+        schema (optional, pyarrow.lib.schema, Metadata, dict):
+            schema to cast the dataframe to during writing
```
Comment (isichei) [inline arrow_pd_parser/export.py:97]: This part of the docstring needs more detail

File: pyproject.toml [hunk 3]
```diff
@@ -18,14 +18,18 @@ repository = "https://github.com/moj-analytical-services/mojap-arrow-pd-parser"
 homepage = "https://github.com/moj-analytical-services/mojap-arrow-pd-parser"
 
 [tool.poetry.dependencies]
-python = ">=3.7.1,<=3.9"
+python = ">=3.7.1,<3.10.0"
 pandas = "^1.2"
-pyarrow = ">=2.0.0,<4.0.0"
+pyarrow = ">=2.0.0,<6.0.0"
```
Comment (isichei) [inline pyproject.toml:23]: Might need to lockdown the lower bound of the pyarrow dependency more (will test this)

**Review by isichei — CHANGES_REQUESTED** [review]
See comments

- Will also need to update readme to reflect changes.
- Also no tests to check the new functions, should be added before merge

File: .version (hunk 1 above)
Comment (s-ducks) [reply .version:1]: this is part of a githoo. `.version` is made in the git hook from the poetry version

Comment (isichei) [reply .version:1]: Yes, but could't this be done to just auto generate the __init__ file?

**Comment by isichei** [comment]
Added in V1 release

**Pull request closed without merging**